railgun/test/integration/udpingress_test.go

```diff
@@ -25,6 +26,9 @@ func TestMinimalUDPIngress(t *testing.T) {
 	if useLegacyKIC() {
 		t.Skip("legacy KIC does not support UDPIngress, skipping")
 	}
+	if dbmode := os.Getenv("KONG_DATABASE"); dbmode != "" && dbmode != "off" {
```

Comment:
this shouldn't be shadowing the global `dbmode` here by making a `getenv`

Reply:
good catch, fixed in 7ce0075aa8e30574d5d759fd3d0574ad4a4456b3 :+1: